### ISSUE TITLE
PULC训练自己的分类模型，生成推理模型时，预测结果不符

### ISSUE THREAD

**Issue by ronghonghui** [PaddlePaddle/PaddleClas#2388]
使用PULC，通过修改person_exists下的yaml文件生成自己的分类模型后，在infer.py和predict_cls.py进行推理都没有问题，但当我使用export_model.py进行转换时，转换后使用paddleclas.PaddleClas加载转换后的模型进行推理时，不仅预测结果有很大的偏差，就连label_names也不对，由于我是0，1二分类，生成的标签变成了'tench, Tinca tinca', 'goldfish, Carassius auratus‘，并不是我原来的标签，但是我是按照https://github.com/PaddlePaddle/PaddleClas/blob/release/2.5/docs/zh_CN/models/PULC/PULC_person_exists.md中6.1.1中的指示进行训练的，请问您知道怎么解决这个问题嘛
训练中使用的yaml参数
# global configs
Global:
  checkpoints: null
  pretrained_model: null
  output_dir: ./output_mask/
  device: gpu
  save_interval: 1
  eval_during_train: True
  eval_interval: 1
  start_eval_epoch: 10
  epochs: 20
  print_batch_step: 10
  use_visualdl: False
  # used for static mode and model export
  image_shape: [3, 256, 128]
  save_inference_dir: ./inference
  # training model under @to_static
  to_static: False
  use_dali: False

# model architecture
Arch:
  name: PPLCNet_x1_0
  class_num: 2
  pretrained: True
  use_ssld: True
  use_sync_bn: True
 
# loss function config for traing/eval process
Loss:
  Train:
    - CELoss:
        weight: 1.0
  Eval:
    - CELoss:
        weight: 1.0

Optimizer:
  name: Momentum
  momentum: 0.9
  lr:
    name: Cosine
    learning_rate: 0.01
    warmup_epoch: 5
  regularizer:
    name: 'L2'
    coeff: 0.00004

# data loader for train and eval
DataLoader:
  Train:
    dataset:
      name: ImageNetDataset
      image_root: ./dataset/mask_data/
      cls_label_path: ./dataset/mask_data/train_list.txt
      transform_ops:
        - DecodeImage:
            to_rgb: True
            channel_first: False
        - ResizeImage:
            size: [256,128]
        - RandFlipImage:
            flip_code: 1
        - TimmAutoAugment:
            prob: 0.0
            config_str: rand-m9-mstd0.5-inc1
            interpolation: bicubic
            img_size: [256,128]
        - NormalizeImage:
            scale: 1.0/255.0
            mean: [0.485, 0.456, 0.406]
            std: [0.229, 0.224, 0.225]
            order: ''
        - RandomErasing:
            EPSILON: 0.0
            sl: 0.02
            sh: 1.0/3.0
            r1: 0.3
            attempt: 10
            use_log_aspect: True
            mode: pixel
    sampler:
      name: DistributedBatchSampler
      batch_size: 64
      drop_last: False
      shuffle: True
    loader:
      num_workers: 8
      use_shared_memory: True

  Eval:
    dataset: 
      name: ImageNetDataset
      image_root: ./dataset/mask_data/
      cls_label_path: ./dataset/mask_data/val_list.txt
      transform_ops:
        - DecodeImage:
            to_rgb: True
            channel_first: False
        - ResizeImage:
            size: [256,128]
        - NormalizeImage:
            scale: 1.0/255.0
            mean: [0.485, 0.456, 0.406]
            std: [0.229, 0.224, 0.225]
            order: ''
    sampler:
      name: DistributedBatchSampler
      batch_size: 64
      drop_last: False
      shuffle: False
    loader:
      num_workers: 4
      use_shared_memory: True

Infer:
  infer_imgs: deploy/images/PULC/person_exists/objects365_02035329.jpg
  batch_size: 10
  transforms:
    - DecodeImage:
        to_rgb: True
        channel_first: False
    - ResizeImage:
            size: [256,128]
    - NormalizeImage:
        scale: 1.0/255.0
        mean: [0.485, 0.456, 0.406]
        std: [0.229, 0.224, 0.225]
        order: ''
    - ToCHWImage:
  PostProcess:
    name: ThreshOutput
    threshold: 0.5
    label_0: mask
    label_1: nomask

Metric:
  Train:
    - TopkAcc:
        topk: [1, 2]
  Eval:
    - TprAtFpr:
    - TopkAcc:
        topk: [1, 2]

使用这个yaml配置文件，是可以进行训练，并且使用infer.py进行推理时正确的，只有转换模型之后才出现错误

**Comment by ChengFengGu** [comment]
同样遇到了这个问题，请问解决了吗

**Comment by ronghonghui** [comment]
时间有点久 记不太清了，我记得好像是哪个配置文件中 图片的长宽参数 是反着的，调整一下就没问题了

**Comment by ronghonghui** [comment]
> 同样遇到了这个问题，请问解决了吗

时间有点久 记不太清了，我记得好像是哪个配置文件中 图片的长宽参数 是反着的，调整一下就没问题了
